builtin/logical/database/backend_test.go

```diff
@@ -585,6 +600,34 @@ func TestBackend_basic(t *testing.T) {
 			t.Fatalf("Creds should not exist")
 		}
 	}
+	assert.Equal(t, 9, len(eventSender.Events))
+	assert.Equal(t, "database/config-write", string(eventSender.Events[0].Type))
```

Comment:
nit: Would be nice to use a for loop to reduce the copy-paste/visual noise here :)